backend/routes/api/resources.js

```diff
@@ -74,6 +75,12 @@ router.get(
       resources = fuse.search(keyword);
     }
 
+    if (tag) {
+      console.log(tag);
+      const fuse = new Fuse(resources, tagOnly);
```

Comment:
How does this work if we search by resources and tag? Right now this will replace the above search.

Reply:
I interpreted it as more of a filter; in this case, I believe that if we search using both a keyword and a tag, it will first search and find all the resources corresponding to the keyword then filter out all the resources in that search that don't have the corresponding tag.

Reply:
Gotcha! Nice—I misunderstood what it was doing at first. Great idea!